platforms/quorum/configuration/deploy-network.yaml

```diff
@@ -74,6 +74,18 @@
       peer: "{{ item.services.peers[0] }}"
     loop: "{{ network['organizations'] }}"
     when: network.type == 'quorum'
+  
+  # This role generates the crypto materials for quorum network based on tessera transaction manager and stores it in vault
+  - name: "Generate crypto material for Tessera Transaction manager"
+    include_role:
+      name: create/generate_crypto_tessera
```

Comment:
folder should be generate/crypto/tessera or create/crypto/tessera

Reply:
I am changing mine to create/crypto/raft